datacube/scripts/spindex.py

```diff
@@ -0,0 +1,189 @@
+# This file is part of the Open Data Cube, see https://opendatacube.org for more information
+#
+# Copyright (c) 2015-2023 ODC Contributors
+# SPDX-License-Identifier: Apache-2.0
+import logging
+import sys
+from typing import Sequence
+
+import click
+import pyproj
+from click import echo, confirm
+from odc.geo import CRS
+
+from datacube.index import Index
+from datacube.ui import click as ui
+from datacube.ui.click import cli
+
+_LOG = logging.getLogger('datacube-system')
+
+
+@cli.group(name='spindex', help='System commands')
+def system():
+    pass
+
+
+@system.command(
+    'create',
+    help='Create unpopulated spatial index for particular CRSes '
+         '(express CRSes as EPSG numbers - e.g. 3857, not "epsg:3857").')
```

Comment:
Does this need to be quite so restrictive?

Reply:
The CRS has to have an EPSG code, or postgis can't use it.  But we could accept more at the CLI, sure.

Reply:
I mostly ask because of the multiple instances of `CRS(f"epsg:{srid}")`. It would be nice to take advantage of the flexibility that odc.geo.CRS affords, especially if in conjuction with something like `normalise_crs`